python/tvm/relay/frontend/onnx.py

```diff
@@ -1065,7 +1062,7 @@ def _impl_v9(cls, inputs, attr, params):
                 scale_w,
                 layout=layout,
                 method=method,
-                align_corners=align_corners,
+                align_corners=False,
```

Comment:
why change this? I thought bilinear + `align_corners=True` makes sense?

Reply:
Upsample was one of the tests that was against our numpy implementation rather than onnxruntime. Checking tvm's output directly against onnx revealed that we were producing incorrect values for bilinear interpolation and always using `align_corners=False` made the outputs match.

Reply:
Interesting. The current ONNX spec says Upsample does only nearest https://github.com/onnx/onnx/blob/master/docs/Operators.md#Upsample, while older versions had `mode` parameter. In either case it doesn't specify what to do with `align_corners`.

Since the spec is dubious and upsample is now deprecated, perhaps we should also warn that upsample is deprecated? Maybe we should always use `method=nearest` as well.

Reply:
Upsample is actually entirely deprecated after opset 9, which did use a mode attribute. That little snippet in the opset 10 documentation is just pointing out that its officially now deprecated.

Reply:
I think its still a good idea to keep it in the importer to support older models though.